README.md

```diff
@@ -19,72 +19,119 @@ The guidelines described below are based on:
 ## Table of Contents
 
 1. [Directory Structure](#directory-structure)
-2. [Modules](#modules)
-3. [Directives and Components](#directives-and-components)
-4. [Services and Dependency Injection](#services-and-dependency-injection)
-5. [Pipes](#pipes)
-6. [Routing](#routing)
-7. [Forms](#forms)
-8. [Reusable libraries](#reusable-libraries)
-9. [Testing](#testing)
-10. [Change Detection](#change-detection)
-11. [TypeScript specific practices](#typescript-specific-practices)
-12. [ES2015 and ES2016 specific practices](#es2015-and-es2016-specific-practices)
-13. [ES5 specific practices](#es5-specific-practices)
-14. [License](#license)
+2. [Directives and Components](#directives-and-components)
+3. [Services and Dependency Injection](#services-and-dependency-injection)
+4. [Pipes](#pipes)
+5. [Routing](#routing)
+6. [Forms](#forms)
+7. [Reusable libraries](#reusable-libraries)
+8. [Testing](#testing)
+9. [Change Detection](#change-detection)
+10. [TypeScript specific practices](#typescript-specific-practices)
+11. [ES2015 and ES2016 specific practices](#es2015-and-es2016-specific-practices)
+12. [ES5 specific practices](#es5-specific-practices)
+13. [License](#license)
 
 ## Directory Structure
 
-* Create folders named for the feature they represent. When a folder grows to contain more than 7 files, start to consider creating a folder for them. Your threshold may be different, so adjust as needed.
+* Create folders named for the [bounded context](http://martinfowler.com/bliki/BoundedContext.html) they represent. When a folder grows to contain more than 15 files, start to consider creating a folder for them. Your threshold may be different, so adjust as needed.
+
+  ```
+  .
+  ├── admin
+  │   ├── home-dashboard.component.ts
+  │   ├── home-dashboard.html
+  │   ├── home-dashboard.component.spec.ts
+  │   ├── login.component.ts
+  │   ├── login.component.spec.ts
+  │   ├── admin.ts
+  │   ├── user-management.service.ts
+  │   └── order-management.service.ts
+  ├── common
+  │   ├── components
```

Comment:
A bit tricky here...May be `components` should be called `directives` because that is what components are - directives with views, however, directives could not be considered components.